components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -73,6 +73,7 @@ Change one or more configuration settings to reflect the desired global Data Fee
 - Update the `node_batch_size` setting to change the number of sets of node data sent in each individual batch to your endpoint. The default value is 50 nodes
 - Use the `updated_nodes_only` setting to determine what data to include in each export. The default setting is `true`, which causes the aggregation of only the *changed* data of updated nodes since the last export. Set `updated_nodes_only` to `false` and it aggregates *all* data of updated nodes since the last export
 - To reduce the IP address range for the collected and processed node data, update the `disable_cidr_filter` setting to `false` **and** update the `cidr_filter` setting to cover the required IP address range. For example, you may wish to send only production or test node traffic
+- The `accepted_status_codes` setting is an array of HTTP status codes that the Data Feed Service will treat as success if returned by the 3rd party endpoint. If the status code is not in the `accepted_status_codes` list then an error will be logged
```

Comment:
@davymcaleer Does this setting treat the codes as translating to "success"? Or if any of these codes are returned, that the codes are considered successful?

Reply:
Any of these codes returned will be considered success